app/components/viral/avatar_component.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Viral
+  # Avatar component to display on items first letter
+  class AvatarComponent < Viral::Component
+    attr_reader :initials, :size, :font_styles
+
+    SIZE_DEFAULT = :medium
+    SIZE_MAPPINGS = {
+      small: 'w-8 h-8',
+      medium: 'w-12 h-12',
+      large: 'w-16 h-16'
+    }.freeze
+
+    def initialize(name: nil, colour_string: nil, size: SIZE_DEFAULT, **system_arguments)
+      @name = name
+      @initials = name.chr
+      @colours = generate_hsla_colour(colour_string || name)
+      @size = size
+      @font_styles = if size == :small
+                       'text-lg font-medium'
+                     else
+                       size == :medium ? 'text-2xl font-semibold' : 'text-5xl font-bold'
```

Comment:
**nitpick (blocking):** Could you please change the `:large` font-size to `text-4xl`. Too me that looks more consistent with the ratio in the Projects listing.

e.g.
![image](https://github.com/phac-nml/irida-next/assets/492127/1dc5c065-c785-408d-9801-0de608536294)

![image](https://github.com/phac-nml/irida-next/assets/492127/ba0802ba-ed46-475e-99e3-ab5b9f25e405)
This font size was changed to `text-4xl`

Reply:
Ok, updated in b527c89b2a555c38d05815947128968d69a635ce